man/Makefile

```
@@ -220,10 +222,7 @@ $(foreach cat,$(categories),$(eval $(call category_rule_md,$(firstword $(subst :
 $(HTMLDIR)/grassdocs.css: grassdocs.css
 	$(INSTALL_DATA) $< $@
 
-$(HTMLDIR)/grass_logo.png: grass_logo.png
-	$(INSTALL_DATA) $< $@
-
-$(MDDIR)/source/grass_logo.png: grass_logo.png
+$(HTMLDIR)/grass_logo.svg: grass_logo.png
```

Preserve the original code for HTML.

```suggestion
$(HTMLDIR)/grass_logo.png: grass_logo.png
```

---

Nice catch I just overlooked this earlier.